src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioWorkspace.cs

```diff
@@ -16,7 +17,7 @@ namespace Microsoft.VisualStudio.LanguageServices
     /// </summary>
     public abstract class VisualStudioWorkspace : Workspace
     {
-        private readonly BackgroundCompiler _backgroundCompiler;
+        private BackgroundCompiler _backgroundCompiler;
```

Comment:
I think I'd prefer to leave the object read-only and have a new mutable bool. But that's me.